projects/packages/paypal-payments/src/legacy/class-simple-payments.php

```diff
@@ -0,0 +1,834 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Simple Payments lets users embed a PayPal button fully integrated with wpcom to sell products on the site.
+ * This is not a proper module yet, because not all the pieces are in place. Until everything is shipped, it can be turned
+ * into module that can be enabled/disabled.
+ *
+ * @package automattic/jetpack-paypal-payments
+ */
+
+namespace Automattic\Jetpack\Paypal_Payments;
+
+use Automattic\Jetpack\Assets;
+use Automattic\Jetpack\Current_Plan as Jetpack_Plan;
+use Jetpack;
+use Jetpack_Components;
+use Jetpack_Currencies;
+use Jetpack_Options;
+use WP_Post;
+
+/**
+ * Simple_Payments
+ */
+class Simple_Payments {
+	// These have to be under 20 chars because that is CPT limit.
+
+	const PACKAGE_VERSION = '0.1.0-alpha';
+
+	/**
+	 * Post type order.
+	 *
+	 * @var string
+	 */
+	public static $post_type_order = 'jp_pay_order';
+
+	/**
+	 * Post type product.
+	 *
+	 * @var string
+	 */
+	public static $post_type_product = 'jp_pay_product';
+
+	/**
+	 * Define simple payment shortcode.
+	 *
+	 * @var string
+	 */
+	public static $shortcode = 'simple-payment';
+
+	/**
+	 * Define simple payment CSS prefix.
+	 *
+	 * @var string
+	 */
+	public static $css_classname_prefix = 'jetpack-simple-payments';
+
+	/**
+	 * Which plan the user is on.
+	 *
+	 * @var string value_bundle or jetpack_premium
+	 */
+	public static $required_plan;
+
+	/**
+	 * Instance of the class.
+	 *
+	 * @var Simple_Payments
+	 */
+	private static $instance;
+
+	/**
+	 * Construction function.
+	 */
+	private function __construct() {}
+
+	/**
+	 * Original singleton.
+	 *
+	 * @todo Remove this when nothing calles getInstance anymore.
+	 *
+	 * @deprecated 10.8
+	 */
+	public static function getInstance() { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
+		_deprecated_function( __METHOD__, 'Jetpack 10.7.0', 'Simple_Payments::get_instance' );
+		return self::get_instance();
+	}
```

Comment:
Do we need this in the new class in the package?

Reply:
We don't. I removed it in [83ee3d0](https://github.com/Automattic/jetpack/pull/43413/commits/83ee3d07174550f9d18b4a8654f5644b20fe66f4).